internal/commands/vaultsecrets/secrets/create.go

```diff
@@ -0,0 +1,168 @@
+		Client:        secret_service.New(ctx.HCP, nil),
+	}
+
+	cmd := &cmd.Command{
+		Name:      "create",
+		ShortHelp: "Create a new static secret.",
+		LongHelp: heredoc.New(ctx.IO).Must(`
+		The {{ template "mdCodeOrBold" "hcp vault-secrets secrets create" }} command creates a new static secret under an Vault Secrets App.
+
+		Once the secret is created, it can be read using
+		{{ template "mdCodeOrBold" "hcp vault-secrets secrets read" }} subcommand.
+		`),
+		Examples: []cmd.Example{
+			{
+				Preamble: `Create new secret in Vault Secrets application on active profile:`,
+				Command: heredoc.New(ctx.IO, heredoc.WithPreserveNewlines()).Must(`
+				$ hcp vault-secrets secrets create secret_1 --data-file=tmp/secrets1.txt
+				`),
+			},
+			{
+				Preamble: `Create new secret in Vault Secrets application on active profile with pipe:`,
+				Command: heredoc.New(ctx.IO, heredoc.WithNoWrap()).Must(`
+				$ echo "my super secret" | hcp vault-secrets secrets create secret_2 --data-file=-
+				`),
+			},
+			{
+				Preamble: `Create secret in different Vault Secrets application, not active profile:`,
+				Command: heredoc.New(ctx.IO, heredoc.WithNoWrap()).Must(`
+				$ hcp vault-secrets secrets create secret_3 --app-name test-app --secret_file=/tmp/secrets2.txt
+				`),
+			},
+		},
+		Args: cmd.PositionalArguments{
+			Args: []cmd.PositionalArgument{
+				{
+					Name:          "SECRET_NAME",
+					Documentation: "The name of the secret to create.",
+				},
+			},
+		},
+		Flags: cmd.Flags{
+			Local: []*cmd.Flag{
+				{
+					Name:         "data-file",
+					DisplayValue: "DATA_FILE_PATH",
+					Description:  "Absolute path to the secrets data file.",
+					Value:        flagvalue.Simple("", &opts.SecretFilePath),
+				},
+			},
+		},
+		RunF: func(c *cmd.Command, args []string) error {
+			opts.AppName = appName
+			opts.SecretName = args[0]
+
+			if runF != nil {
+				return runF(opts)
+			}
+			return createRun(opts)
+		},
+	}
+
+	return cmd
+}
+
+type CreateOpts struct {
+	Ctx     context.Context
+	Profile *profile.Profile
+	IO      iostreams.IOStreams
+	Output  *format.Outputter
+
+	AppName              string
+	SecretName           string
+	SecretValuePlaintext string
+	SecretFilePath       string
+	PreviewClient        preview_secret_service.ClientService
+	Client               secret_service.ClientService
+}
+
+func createRun(opts *CreateOpts) error {
+	if err := readPlainTextSecret(opts); err != nil {
+		return err
+	}
+
+	req := secret_service.NewCreateAppKVSecretParamsWithContext(opts.Ctx)
+	req.LocationOrganizationID = opts.Profile.OrganizationID
+	req.LocationProjectID = opts.Profile.ProjectID
+	req.AppName = opts.AppName
+
+	req.Body = secret_service.CreateAppKVSecretBody{
+		Name:  opts.SecretName,
+		Value: opts.SecretValuePlaintext,
+	}
+
+	resp, err := opts.Client.CreateAppKVSecret(req, nil)
+	if err != nil {
+		return fmt.Errorf("failed to create secret with name: %s - %w", opts.SecretName, err)
```

Comment:
```suggestion
		return fmt.Errorf("failed to create secret with name %q: %w", opts.SecretName, err)
```

Use pattern of <wrapped error message>: <error>